perf/perftool.py

```diff
@@ -39,9 +38,19 @@ def setUp(self):
 
         # Check for basic utilities
         smm = SoftwareManager()
-        for package in ['gcc', 'make', 'perf']:
+        detected_distro = distro.detect()
+        kernel_ver = platform.uname()[2]
+        deps = ['gcc', 'make']
+        if 'Ubuntu' in detected_distro.name:
+            deps.extend(['linux-tools-common', 'linux-tools-%s' % kernel_ver])
+        elif detected_distro.name in ['redhat', 'SuSE', 'fedora']:
```

Comment:
i think this will work on centos as well can we have check for that as well or it can like 

it can be be only 
```
if ubuntu:

else :
```